coderadar-test/src/main/asciidoc/admin.adoc

```diff
@@ -12,89 +12,85 @@ v{version}, {date}
 
-Coderadar makes use of http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready[Spring Boot Actuator],
-which provides a mechanism to expose certain metrics via HTTP. A list of the monitoring endpoints can by default be accessed via
-"/actuator" on each server node in a coderadar cluster.
+The following properties control how long a access/refresh token is valid:
 
-For an explanation of the endpoints and monitoring metrics provided by Spring Boot Actuator, please
-consult the manual.
+`coderadar.authentication.accessToken.durationInMinutes=15`
+`coderadar.authentication.refreshToken.durationInMinutes=86400`
 
-=== Coderadar-specific Monitoring Metrics
+==== Configuring authentication
 
-Coderadar exposes the following monitoring metrics as a JSON object via the "/metrics" endpoint.
-All metrics starting with the prefix `coderadar.` are coderadar-specific metrics.
+Authentication can be enabled or disabled with the following property:
 
-.Coderadar Monitoring Metrics
-|===
-|Metric name |Description
+`coderadar.authentication.enabled=true`
 
-| coderadar.CommitAnalyzer.commits.*
-| Count and throughput of commits having been analyzed.
+==== Configuring CORS
 
-| coderadar.CommitAnalyzer.files.*
-| Count and throughput of files having been analyzed.
+CORS can be enabled or disabled with the following property:
 
-| coderadar.CommitMetaDataScanner.commits.*
-| Count and throughput of commits having been scanned for metadata.
+`coderadar.cors.enabled=true`
 
-| coderadar.CommitToFileAssociator.commits.*
-| Count and throughput of commits whose files have been associated with it.
+==== Configuring logging levels
 
-| coderadar.CommitToFileAssociator.files.*
-| Count and throughput of files having been associated with a commit.
+The logging levels can be adjusting with the following properties:
 
-| coderadar.FileMetadataScanner.commits.count.*
-| Count and throughput of commits whose files have been scanned for metadata.
+`logging.level.io.reflectoring.coderadar=DEBUG`
+`logging.level.org.reflections=ERROR`
 
-| coderadar.FileMetadataScanner.files.*
-| Count and throughput of files having been scanned for metadata.
+==== Configuring the log file
 
-| coderadar.jobs.acquisitionConflicts.*
-| Count and rate of conflicts between server nodes who tried to acquire the same
-  job for processing. It's OK if this happens and the `coderadar.jobs.inProgress`
-  counter stays at approximately the number of coderadar slaves in the cluster.
-  If the `coderadar.jobs.inProgress` counter falls significantly below the number
-  of coderadar slaved in the cluster, they are probably contesting too much for
-  the jobs waiting in queue.
+The log file's path can be adjusted with the following property:
 
-| coderadar.jobs.failed.*
-| Count and rate of failed jobs.
+`logging.file=coderadar.log`
 
-| coderadar.jobs.finished.*
-| Count and throughput of successfully finished jobs.
+==== Configuration the database connection
```

Comment:
Either **Configuring the database connection** or **Configuration of the database connection**.

Reply:
Done